lib/capistrano/tasks/assets.rake

```diff
@@ -62,15 +62,29 @@ namespace :deploy do
       end
     end
 
+    def manifest_path(filename)
+      release_path.join('public', fetch(:assets_prefix), filename)
+    end
+
+    def with_backup_and_manifest_paths &block
+      backup_path = release_path.join('assets_manifest_backup')
+      manifest_path = test(:ls, manifest_path('.sprockets-manifest*'))
```

Comment:
Just use it as if condition, not save it to var. Most capistrano code use test result directly in if condition, as you can see even in this file.